zap/logger_test.go

```diff
@@ -197,6 +294,11 @@ func TestCoreOption(t *testing.T) {
 	assert.True(t, strings.Contains(buf.String(), "this is a warn log"))
 	// test console encoder result
 	assert.True(t, strings.Contains(buf.String(), "\tERROR\t"))
+
+	logger.SetLevel(hlog.LevelWarn)
```

Comment:
for this situation level should from high -> low, here info -> warn, error will always logged, and SetOutput not covered